InstaApp/InstaApp/Models/LikedComment.swift

```diff
@@ -0,0 +1,30 @@
+//
+//  LikedComment.swift
+//  InstaApp
+//
+//  Created by Валентин Коскин on 20.10.2022.
+//
+
+import Foundation
+
+/// Describe data in NoiceCell
+struct LikedComment {
+    private let text: String
+    private let senderPhotoName: String
+    private let likedPhotoName: String
+    private let date: String
+    private let userName: String
+
+    init(text: String, senderPhotoName: String, likedPhotoName: String, date: String, name: String) {
+        self.text = text
+        self.senderPhotoName = senderPhotoName
+        self.likedPhotoName = likedPhotoName
+        self.date = date
+        self.userName = name
```

Comment:
если имена не совпадают self не нужен
```suggestion
       userName = name
```